### PR TITLE
Add Express5 alpha mock

### PR DESCRIPTION
**Pull request by cristiancavalli**
- Existing Express error extraction machinery
    works with Express5 in its current state.
  - Add a server mock so that compatibility can
    be tested over time.
  - Add `npm-install-version` as development dependency
    to manage Express5 usage in server mock.

### CODE REVIEW

**Comment by cristiancavalli** [comment]
@GoogleCloudPlatform/node-team PTAL

File: tests/test-servers/express_5_scaffold_server.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+var app = express();
+var errorHandler = require('../../index.js')({
+  ignoreEnvironmentCheck: true
+});
+var bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+
+app.post(
+  '/testErrorHandling'
```
Comment (ofrobots) [inline tests/test-servers/express_5_scaffold_server.js]: nit: this file doesn't seem to follow the style we normally use.

Comment (ofrobots) [inline tests/test-servers/express_5_scaffold_server.js]: actually all of the other files in the directory don't follow the style either, so can be addressed in a follow-on PR.

**Comment by DominicKramer** [comment]
Any further work in this area will be done in the [google-cloud-node/error-reporting](https://github.com/GoogleCloudPlatform/google-cloud-node/tree/master/packages/error-reporting) repository and is being tracked by issue [2294](https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2294) in that repository.

**Pull request closed without merging**